web/src/data/firebase/authentication.js

```diff
@@ -0,0 +1,33 @@
+import firebase from "firebase";
+import {User} from "core/entities";
+
+export const signInWithEmailAndPassword = async  (email,password) => {
```

Comment:
![image](https://user-images.githubusercontent.com/823150/49802408-1ae00c00-fd4d-11e8-84a2-bc73369d2882.png)
Not big deal, mas seria legal se o código fosse consistente.
ou usa espaco deopis de virgula ou nao usa. Tu podes configurar o prettier na tua IDE Ttbm